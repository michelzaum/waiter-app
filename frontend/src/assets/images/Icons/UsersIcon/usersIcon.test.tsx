import { render } from '@testing-library/react';
import { describe, it, expect } from 'vitest';
import { UsersIcon } from '.';

describe('UsersIcon', () => {
  it('Should render the UsersIcon', () => {
    //given
    const result = render(<UsersIcon />);

    // then
    expect(result).toMatchSnapshot();
  });
});
