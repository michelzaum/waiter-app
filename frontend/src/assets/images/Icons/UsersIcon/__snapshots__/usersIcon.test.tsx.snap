// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`UsersIcon > Should render the UsersIcon 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <svg
        fill="none"
        height="25"
        viewBox="0 0 24 25"
        width="24"
        xmlns="http://www.w3.org/2000/svg"
      >
        <circle
          cx="7.99835"
          cy="9.00854"
          r="3.49145"
          stroke="currentColor"
          stroke-linecap="round"
          stroke-linejoin="round"
          stroke-width="1.5"
        />
        <circle
          cx="17.0021"
          cy="9.99897"
          r="2.50104"
          stroke="currentColor"
          stroke-linecap="round"
          stroke-linejoin="round"
          stroke-width="1.5"
        />
        <path
          d="M1.99585 20.5033V19.4859C1.99585 17.285 3.77959 15.5012 5.98051 15.5012H10.0162C12.2171 15.5012 14.0008 17.285 14.0008 19.4859V20.5033"
          stroke="currentColor"
          stroke-linecap="round"
          stroke-linejoin="round"
          stroke-width="1.5"
        />
        <path
          d="M17.0021 15.5012H18.1045C20.3055 15.5012 22.0892 17.285 22.0892 19.4859V20.5033"
          stroke="currentColor"
          stroke-linecap="round"
          stroke-linejoin="round"
          stroke-width="1.5"
        />
      </svg>
    </div>
  </body>,
  "container": <div>
    <svg
      fill="none"
      height="25"
      viewBox="0 0 24 25"
      width="24"
      xmlns="http://www.w3.org/2000/svg"
    >
      <circle
        cx="7.99835"
        cy="9.00854"
        r="3.49145"
        stroke="currentColor"
        stroke-linecap="round"
        stroke-linejoin="round"
        stroke-width="1.5"
      />
      <circle
        cx="17.0021"
        cy="9.99897"
        r="2.50104"
        stroke="currentColor"
        stroke-linecap="round"
        stroke-linejoin="round"
        stroke-width="1.5"
      />
      <path
        d="M1.99585 20.5033V19.4859C1.99585 17.285 3.77959 15.5012 5.98051 15.5012H10.0162C12.2171 15.5012 14.0008 17.285 14.0008 19.4859V20.5033"
        stroke="currentColor"
        stroke-linecap="round"
        stroke-linejoin="round"
        stroke-width="1.5"
      />
      <path
        d="M17.0021 15.5012H18.1045C20.3055 15.5012 22.0892 17.285 22.0892 19.4859V20.5033"
        stroke="currentColor"
        stroke-linecap="round"
        stroke-linejoin="round"
        stroke-width="1.5"
      />
    </svg>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
